docs/development/documentation_preview.rst

```diff
@@ -4,13 +4,20 @@
 Documentation Preview
 *********************
 
-Most of the time it's enough to build the documentation locally and see how things are going. But sometimes 
-it's nice to share our changes and get some feedback from other collaborators. 
+To preview your changes to the documentation please:
 
-To preview your changes to the documentation first make sure you have GitHub Actions activated on your fork
-(see the Actions tab). Then, there are two ways to trigger the build:
+#. Enable GitHub Actions to run on your fork (see the *Actions* tab).
+#. Make sure your fork is deploying GitHub Pages inside the root of ``gh-pages`` branch (go to *Settings* -> *Pages*).
```

Comment:
can we make this

Under *Settings* -> *Pages* in your fork, make sure GitHub Pages is being built from the ```gh-pages``` branch and the ```/ (root)``` folder.